napari/viewer.py

```diff
@@ -38,6 +40,7 @@ def __init__(
         order=(),
         axis_labels=(),
         show=True,
+        welcome=True,
```

Comment:
folks have been pretty protective of Viewer params, so just flagging this one for review.  I think the welcome widget is innocuous enough that it could just be on all the time...

Reply:
I needed a way to disable this so screenshot tests would not fail. That is the main reason I added it.

But I am open for suggestions :)

Reply:
yeah, i think i'd drop here - fine to leave on `Window` though, in case we want to expose later

Reply:
Will fix.

Reply:
> I needed a way to disable this so screenshot tests would not fail. That is the main reason I added it.

ah yeah, we should fix those! they were a bit of a hack!! really the tests are just changing that the screen was black before hand, maybe now they should test that they see the welcome visual? they could also check that the before/ after screenshots were not equal. Probably many ways that can be improved!! Could be done in this PR too or followup. I can help advise if needed

Reply:
I fixed the tests, so that no assumption on the welcome screen is made now (I disable it on tests)

I can add a tests that is specific for it.

Reply:
>Could be done in this PR too or followup. I can help advise if needed

Let's do that so, opened https://github.com/napari/napari/issues/2548 to track it